docs/control-structures.rst

```diff
@@ -65,11 +65,11 @@ uses up at least one stack slot and there are only 1024 slots available.
 External Function Calls
 -----------------------
 
-The expressions ``this.g(8);`` and ``c.g(2);`` (where ``c`` is a contract
-instance) are also valid function calls, but this time, the function
-will be called "externally", via a message call and not directly via jumps.
-Please note that function calls on ``this`` cannot be used in the constructor,
-as the actual contract has not been created yet.
+Functions can also be called using ``this.g(8);`` and ``c.g(2);`` notation, where 
+c is a contract instance and g is a function belonging to c. Calling 
+function g via either method results in it being called "externally", via a message
+call and not directly via jumps. Please note that function calls on ``this`` cannot
+be used in the constructor, as the actual contract has not been created yet.
```

Comment:
Starting a new sentence always on a new line causes less rewrapping and gives better diffs (it would for example make it easier to see that you did not modify the last sentence):
```suggestion
Functions can also be called using ``this.g(8);`` and ``c.g(2);`` notation, where 
c is a contract instance and g is a function belonging to c.
Calling function g via either method results in it being called "externally", via a
message call and not directly via jumps.
Please note that function calls on ``this`` cannot be used in the constructor,
as the actual contract has not been created yet.
```